@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -531,6 +532,15 @@ export class OmvDecodedTileEmitter implements IOmvEmitter {
     ): void {
         this.processFeatureCommon(env);
 
+        const { projection, center, tileKey } = this.m_decodeInfo;
+
+        // get the bounds, and the size of the OMV Tile in its webmercator space.
+        const tileDataBounds = new THREE.Box3();
```

Comment:
The names ``tileDataBounds`` and ``tileDataSize`` (specially this last one) are a bit confusing, since it seems like you're taking about array ranges and sizes (i.e. bytes), while you're talking about meassurements. Why renamig to this?